app/lib/discount.rb

```diff
@@ -0,0 +1,31 @@
+module Discount
+
+  def apply_discounts
+    subtotal - threshold_discount - item_discount('Muffin')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.